smo/utils/templating.py

```diff
@@ -105,7 +104,6 @@ def render_template(template_path: str, macro_roots: Optional[List[str]] = None,
 
         # Create the environment and add the basic filters
         new_env: Environment = Environment(loader=loader, trim_blocks=True)
-        new_env.filters['qtLiteral'] = qt_literal
```

Comment:
I dont see it being used anywhere in code except tests which are deleted so this seem fine. Just confirm once if it is getting used anywhere, otherwise changes look good.